packages/client/src/sync/fetcher/blockfetcher.ts

```diff
@@ -86,7 +86,11 @@ export class BlockFetcher extends BlockFetcherBase<Block[], Block> {
       }
       // Supply the common from the corresponding block header already set on correct fork
       const block = Block.fromValuesArray(values, { common: headers[i].common })
-      await block.validateData()
+      // Only validate the data integrity
+      // Upon putting blocks into blockchain (for BlockFetcher), `validateData` is called again
+      // In ReverseBlockFetcher we do not need to validate the entire block, since CL
+      // expects us to sync with the requested chain tip header
+      await block.validateDataIntegrity()
```

Comment:
Is it really necessary to add a wrapper function that's only used in one place?  Feels like we should just call `block.verifyData` and then in the comments explan why we're not validating transactions.

Reply:
(Or is there a somewhat strong reasoning (which might be) that the API is better off with (yet) another explicit validation method?

Reply:
Not really a strong reason to do this. I will remove the method and directly call into `verifyData`!

Reply:
Have addressed